main.nf

```diff
@@ -33,6 +33,7 @@ def helpMessage() {
       --proteins [file]                     Path to FASTA file containing protein sequences
 
     Pipeline options:
+      --show_supported_models [str]         Writes out supported models. Does not run actual prediction pipeline. Do not specify this in combination with other input files. Default: false.
```

Comment:
Maybe "Print out supported models." ? Do we need the warning that it should not be combined with input data?